magento/pwa-studio
# Create i18n extension and add french pack

## Description

Uses the [previous example](https://github.com/magento-research/pwa-studio-target-experiments/pull/2/files) to construct an extension that provides some maintainable translation packs.

## Related Issue





Closes [PWA-1071](https://jira.corp.magento.com/browse/PWA-1071).

## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps



1. Delete `venia-ui/i18n/fr_FR.json` if you have it. 
2. `yarn venia add -D @magento/venia-sample-language-packs@0.0.1`
3. `yarn watch:venia` using a backend with a french locale enabled.
4. Use the store switcher to change to French store view. Translations should work.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
